ui/DesignSystem/Component/Registration/NavigationButtons.svelte

```diff
@@ -0,0 +1,36 @@
+<script>
+  import { Button } from "../../Primitive";
+
+  export let onCancel = null;
```

Comment:
An interface for this component which would be general and meaningful could look like this:

``` svelte
    <NavigationButtons
      style="margin-top: 32px;"
      cancelLabel={$cancelLabel}
      submitLabel={$nextButtonLabel}
      on:cancel={cancel}
      on:submit={next} />
```

Where both labels have sensible defaults.